pytarallo/ProductToUpload.py

```diff
@@ -0,0 +1,9 @@
+from pytarallo.Product import Product
+
+
+class ProductToUpload(Product):
+    """Not sure what this is"""
```

Comment:
Uhm... can we maybe clarify this a little bit better? Calling in @lvps for extra support

Reply:
Well, it's a product and it should be uploaded asd
Can you make it inherit from the Product class, so it's the same thing with a different name? IIRC there's no difference right now, but there may be in the future, at least we're prepared, especially in code that will use this library (i.e. we don't have to review every use of Product to check if it should be ProductToUpload when they start to differ in some way)